### ISSUE TITLE
Last.fm Scrobbling Broken? 

### ISSUE THREAD

**Issue by Scotty-Trees** [Taiko2k/TauonMusicBox#400]
I don't know if this is a bug or a feature, as I'm not sure exactly how Tauon handles Last.fm scrobbling, but when I check my last.fm profile web page, none of my recent tracks from the last couple of days I've played through Tauon have showed up. I do also have Spotify (flatpak) installed and all of those tracks have been scrobbled normally. Last working Tauon track that was scrobbled was Oct 1 or 2'nd, but nothing since than. I don't know if somehow a recent flatpak runtime update has changed anything. I've taken a few attempts to see if I could figure this out, but nothing I can tell since I'm definitely no expert. I have tried to restart my system, restart Tauon, log out and log back in of the Last.fm menu in Tauon, have only Tauon open when playing music ie not running Spotify in case it possibly somehow were to conflict with each other (seems unlikely though that's the case), and I've also run Tauon from the terminal, but I don't appear to see anything error wise or last.fm wise. I do have scrobbling set as active in the Tauon menu so I know it's at least ON. Hope some of that helps target where to go next from here!

I do have on question though. I assume last.fm scrobbles in real time, so Tauon should show that data to last.fm's web page after a track is played. Is Tauon supposed to function that way or does it cache a bunch of music tracks and then sends them off to last.fm at a specified time or something? That's about all I can think of at the moment. Hopefully this is some sort of issue and I'm just not one big giant fool :P 

Below is the output of running Tauon from the terminal where I let a few tracks play through in their entirely and a few screenshots to show as far as I can tell, Last.fm is on in Tauon:

```
scott@pop-os:~$ flatpak run com.github.taiko2k.tauonmb
Tauon Music Box v6.3.2
Copyright 2015-2020 Taiko2k captain.gxj@gmail.com

Gtk-Message: 05:37:17.809: Failed to load module "appmenu-gtk-module"
Detected running as Flatpak
Installed as Snap
User files location: /home/scott/.var/app/com.github.taiko2k.tauonmb/data/TauonMusicBox
Found XDG-Downloads: /home/scott/Downloads
Found XDG-Music: /home/scott/Music
Install directory: /app/bin
Database loaded in 0.081 seconds.
Applying scale based on xft setting
Using UI scale: 1.0
Could not connect to gnome media keys
Start PHAzOR backend...
ph: PHAzOR starting up
Using SDL version: 2.0.12
Applying theme: Carbon
Auto save playtime
Transition gapless mode
pa: End of stream
pa: Read next gapless
pa: Reset position counter
pa: End of stream
Transition gapless mode
pa: Next registered while buffer was draining
pa: -- remaining was 74334
pa: End of stream
pa: Read next gapless
pa: Reset position counter
Auto save playtime
pa: End of stream
Transition gapless mode
pa: Next registered while buffer was draining
pa: -- remaining was 84877
pa: End of stream
pa: Read next gapless
pa: Reset position counter
Transition gapless mode
pa: End of stream
pa: Read next gapless
pa: Reset position counter
Transition gapless mode
pa: End of stream
pa: Read next gapless
pa: Reset position counter
```
![Screenshot from 2020-10-05 05-44-10](https://user-images.githubusercontent.com/45135187/95071920-08785480-06d0-11eb-8905-40681eb9d4e5.png)
![Screenshot from 2020-10-05 05-43-56](https://user-images.githubusercontent.com/45135187/95071964-19c16100-06d0-11eb-9064-b48a234422be.png)

**Comment by Taiko2k** [comment]
Tauon is supposed to scrobble after when the track ends or stops.

That settings page looks correct. Sounds like it should be working...

Which backend do you have selected in settings, GSreamer or Phazor?

**Comment by Scotty-Trees** [comment]
Ah I think you might've already nailed it! I was using the GStreamer backend, but then a few days ago on another bug post [#381](https://github.com/Taiko2k/TauonMusicBox/issues/381) I was having a similar issue as well, so I have actually been using your Phazor backend of the last couple of days, which works pretty great btw. But I just read your edit and you're aware that Phazor and Last.fm don't work together. I hope that's something to tackle in the future? Because I've honestly been pretty happy with Phazor even though I know it's still in development. 

I guess if I want to scrobble I'll just have to stick to the GStreamer backend for now! But if Phazor ever supports last.fm I'm onboard! Thanks again for the quick turn around. I'd say this issue is resolved, though I may have to open a few for Phazor haha just kidding man! :D

**Comment by Taiko2k** [comment]
Yep its already fixed in master. I plan on doing a new release within the next week.

**Comment by Scotty-Trees** [comment]
I'm stoked! New versions are always a great welcome, wooo!

**Comment by Taiko2k** [comment]
Okay, a new version of Tauon has been released (v6.3.3)

**Comment by Scotty-Trees** [comment]
Just updated to v6.3.3, and tested GStreamer and Phazor, both successfully submit to last.fm! Thank you kindly for the super fast fix. I'll definitely keep an eye on any Phazor progress and updates for sure I'm liking it very much, cheers mate :)
